smoke-tests/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM mcr.microsoft.com/playwright:v1.45.0-jammy
+
+
+# Set the working directory
+WORKDIR /usr/src/app
+
+# Copy your project files into the Docker container
+COPY . .
```

Comment:
you might copy some temporary directories such as `node_modules`. I would copy the minimum you need to prepare the deps. So I think `package.json` should be enough, isn't it?